fbw-a380x/src/systems/systems-host/systems/LegacyFuelInit.ts

```diff
@@ -0,0 +1,6 @@
+export class LegacyFuelInit {
+  init() {
+    const fuelWeight = SimVar.GetSimVarValue('FUEL TOTAL QUANTITY WEIGHT', 'kilograms');
+    SimVar.SetSimVarValue('L:A32NX_FUEL_DESIRED', 'kilograms', fuelWeight);
+  }
+}
```

Comment:
```suggestion
export class LegacyFuelInit implements Instrument {
  init() {
    const fuelWeight = SimVar.GetSimVarValue('FUEL TOTAL QUANTITY WEIGHT', 'kilograms');
    SimVar.SetSimVarValue('L:A32NX_FUEL_DESIRED', 'kilograms', fuelWeight);
  }
  
  public onUpdate(): void {
    // noop
  }
}
```

Then just add to the backplane in index.ts. No need to micro-manage init/update of individual components.

Reply:
Maybe also add a comment explaining the purpose of this.